tools/dev.sh

```diff
@@ -118,7 +118,7 @@ user_ui_pid=$!
 
 (
   for _ in {1..60}; do # ~1 minute timeout
-    if curl -s --head http://localhost:8080/ | head -n 1 | grep "200 OK" > /dev/null; then
+    if curl -s -H 'User-Agent: Mozilla/5.0' --head http://localhost:8080/unknown | head -n 1 | grep "200 OK" > /dev/null; then
```

Comment:
Just checked and you don't actually need the header for this to work, since we have `-s --head` specified.

Let's also switch this to check for the .ico file served by the user UI instead since I think it's probably a bad idea to depend on `/<agent>` returning a 200 when you try to GET an agent that doesn't exist; I suspect that behavior is actually a bug, we probably want some splash page there that tells you that an admin needs to configure some agents.

```suggestion
    if curl -s --head http://localhost:8080/favicon.ico | head -n 1 | grep "200 OK" > /dev/null; then
```